package.json

```diff
@@ -0,0 +1,148 @@
+{
+	"name": "virtual-comments",
+	"displayName": "Virtual Comments",
+	"publisher": "virtual-comments",
+	"description": "Virtual Comments",
+	"icon": "resources/logo.png",
+	"version": "0.0.3",
+	"private": true,
+	"license": "MIT",
```

Comment:
Project license is file is Apache and we have mentioned here MIT. Please correct it.